src/scf-release/src/acceptance-tests-brain/test-scripts/minibroker_helper.rb

```diff
@@ -98,7 +98,8 @@ def run_test
 
             run "kubectl get namespace #{minibroker_namespace} 2> /dev/null || kubectl create namespace #{minibroker_namespace}"
             run "helm init --client-only"
-            run(*%W(helm upgrade #{helm_release} minibroker
+            run_with_retry 30, 5 do
+              run(*%W(helm upgrade #{helm_release} minibroker
```

Comment:
Technically this messes up the 4-space indent, but I don't know if we care.

Reply:
Fixed with commit 81bb17e4880e10546b03f94f12ec3fcf3a80c3b0.
(My emacs seems to use 2-space indent, and there is nothing to tell it otherwise).